eng/pipelines/coreclr/jitstress-isas-avx512.yml

```diff
@@ -1,20 +1,6 @@
 # This pipeline only runs on GitHub PRs, not on merges.
 trigger: none
 
-# Only run on specific changes to the JIT directory that are likely to affect AVX-512.
-pr:
-  branches:
-    include:
-    - main
-  paths:
-    include:
-    - src/coreclr/jit/hwintrinsiccodegenxarch.cpp
-    - src/coreclr/jit/hwintrinsiclistxarch.h
-    - src/coreclr/jit/hwintrinsicxarch.cpp
-    - src/coreclr/jit/instrsxarch.h
-    - src/coreclr/jit/emitxarch.cpp
-    - src/coreclr/jit/emitxarch.h
-
```

Comment:
Removed this after a chat with @tannergooding. The path filters here prevent manual trigger of this pipeline if the changed files don't match.

Since AVX-512 support has been stable for a while, this likely doesn't need to run for all PRs touching these files anymore. The weekly scheduled should catch any regressions.

In any case, editing the pipeline triggered a run, and it is now passing.

Reply:
> The weekly scheduled should catch any regressions.

And can be manually triggered, just like the other ISA stress jobs if we know we're touching it otherwise.